src/Range.ts

```diff
@@ -0,0 +1,53 @@
+import SmartRange from "./SmartRange";
```

Comment:
No licence information. Not necessary but you provided it for other files.

You should also include the licence information / MIT licence in the project directory

Reply:
duplicated